pygmt/src/grdfilter.py

```diff
@@ -54,11 +54,23 @@ def grdfilter(grid, **kwargs):
         The name of the output netCDF file with extension .nc to store the grid
         in.
     filter : str
-        ``xwidth[/width2][modifiers]``.
-        Name of filter type you which to apply, followed by the width
-        b: Box Car; c: Cosine Arch; g: Gaussian; o: Operator; m: Median;
-        p: Maximum Likelihood probability; h: histogram
-        Example: F='m600' for a median filter with width of 600
+        **b**\|\ **c**\|\ **g**\|\ **o**\|\ **m**\|\ **p**\|\ **h**\ *xwidth*\
+        [/*width2*\][*modifiers*].
+        Name of filter type you which to apply, followed by the width:
+
+        b: Box Car
+
+        c: Cosine Arch
+
+        g: Gaussian
+
+        o: Operator
+
+        m: Median
+
+        p: Maximum Likelihood probability
+
+        h: histogram
```

Comment:
```suggestion
        - **b** - Box Car
        - **c** - Cosine Arch
        - **g** - Gaussian
        - **o** - Operator
        - **m** - Median
        - **p** - Maximum Likelihood probability
        - **h** - Histogram
```

Reply:
When I was testing it in the Sphinx previewer it put all of the bullet points on the same line if there wasn't a blank line in between. Will this suggestion work?

Reply:
Yes, there just needs to be a blank line in between the paragraph before ("...followed by the width") and the list in order for it to work. See L46 in pygmt/helpers/decorators.py for an example.

Reply:
> When I was testing it in the Sphinx previewer it put all of the bullet points on the same line if there wasn't a blank line in between. Will this suggestion work?

In thinking about it more, you probably needed lines in between all the bullet points because there wasn't anything before the b, c,... to denote a list (e.g., ``` - b```). If you ever need to specify that text should be separated into multiple lines without being formatted as a bullet list, you can use the ``|`` symbol as an alternative to having lines in between.

Reply:
I'm going to leave it as is for now, but will try and work on bullet list formatting in the near future.